store-and-retrieve-data/use-storagehub-sdk/create-a-bucket.md

```diff
@@ -74,120 +39,155 @@ async function run() {
 await run();
 ```
 
---8<-- 'text/store-and-retrieve-data/use-storagehub-sdk/initialize-clients-summary.md'
+## Check MSP Health
 
+Next, since you are already connected to the MSP client, check its health status before creating a bucket.
 
-## Connect to the MSP Client
+1. Replace the placeholder `// **PLACEHOLDER FOR STEP 1: CHECK MSP HEALTH**` with the following code:
 
-Next, you'll need to connect to the MSP client and check its health status before creating a bucket.
+    ```ts title="index.ts // **PLACEHOLDER FOR STEP 1: CHECK MSP HEALTH**"
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:check-msp-health'
+    ```
 
-Replace the placeholder `// **PLACEHOLDER FOR STEP 1: CONNECT TO MSP CLIENT & CHECK HEALTH**` with the following code:
+2. Check the health status by running the script:
 
-```ts title="// **PLACEHOLDER FOR STEP 1: CONNECT TO MSP CLIENT & CHECK HEALTH**"
---8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/create-a-bucket.ts:connect-msp-client'
-```
+    ```bash
+    ts-node index.ts
+    ```
 
-Then, check the health status by running the script:
+    The response should return a **`healthy`** status, like this:
 
-```bash
-ts-node index.ts
-```
+    --8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/output-01.html'
 
-The response should return a **`healthy`** status, like this:
+## Create a Bucket
 
---8<-- 'code/store-and-retrieve-data/use-storagehub-sdk/create-a-bucket/output-01.html'
+To create a bucket, you are going to: 
 
-## Initialize the StorageHub Client 
+1. Create a `getValueProps` helper method within `mspService.ts`.
+2. Create a `createBucket` helper method within `bucketOperations.ts`.
+3. Update the `index.ts` file to trigger the logic you've implemented.
```

Comment:
I feel like this just needs a tiny bit of context, not a lot as it's discussed in the following section, but enough to fill in the gaps. If you put yourself in the shoes of someone who has never created a bucket before, it's like why do I need value props, what am I doing with that?